.eslintrc.json

```diff
@@ -12,11 +12,7 @@
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "extends": [
-    "airbnb",
-    "plugin:react/recommended",
-    "plugin:react-hooks/recommended"
-  ],
+  "extends": ["airbnb", "plugin:react/recommended"],
```

Comment:
- [ ] Kindly don't edit linters, linters help us to keep standards and best practices, if errors arise let's try to fix them 👍.

Reply:
- [ ] @Nezerwa as previosuly requested kindly return the linter configuration to its original state ` "extends": ["airbnb", "plugin:react/recommended", "plugin:react-hooks/recommended"],`